react-search-component/my-app/src/SearchableList Components/SearchBar.tsx

```diff
@@ -0,0 +1,25 @@
+import { ChangeEvent } from 'react';
+
+type SearchBarProps = {
+  onSearch: (searchText: string) => void;
+};
+
+export default function SearchBar({ onSearch }: SearchBarProps) {
+  function handleOnChange(event: ChangeEvent<HTMLInputElement>) {
+    onSearch(event.target.value);
+  }
+
+  return (
+    <div>
+      <label>
+        <input
+          onChange={handleOnChange}
```

Comment:
This is simple enough that you could just inline `handleOnChange`. Less typing, fewer imports.